lib/content-services/src/lib/group/services/group.service.ts

```diff
@@ -36,6 +36,7 @@ export class GroupService {
     }
 
     async listAllGroupMembershipsForPerson(personId: string, opts?: any, accumulator = []): Promise<GroupEntry[]> {
+        return Promise.resolve([]);
```

Comment:
I didn't implement any of the user / group services but this would likely be delegated to IdP API calls